test/test_bson.py

```diff
@@ -1053,6 +1054,73 @@ def test_unicode_decode_error_handler(self):
         self.assertRaises(InvalidBSON, decode, invalid_both, CodecOptions(
             unicode_decode_error_handler="junk"))
 
+    def test_regex_pickling(self):
+        dbr = Regex(".?")
+        pickled_with_3 = b'\x80\x04\x959\x00\x00\x00\x00\x00\x00\x00\x8c\n' \
```

Comment:
Please wrap the multi-line string with () instead of `\`

Reply:
Done.